### PR TITLE
Show pointer cursor to admins only

### LINKED ISSUE #3101: PDP shows "pointer" cursor for non-admins when hovering over PDP image
The PDP _should_ show a cursor for the admin, as there is an action that the admin can take by clicking on the PDP image gallery.

The PDP _should not_ show a cursor for the customer (until #3100) because there is no action that can be taken by clicking.

Related: #2469

### PR DESCRIPTION
**Pull request by codejockie**
Resolves #3101 

This PR ensures that the pointer cursor is only visible to admins on the PDP image and also ensures the hover effect is not visible to non-admins.

## Test
1. Login as admin in a browser
2. Open the app in another browser either by logging in as a non-admin or without logging in
3. Click on a product with image set in both browsers
4. On the browser you're logged in as an admin hover the mouse the product's image, confirm you see a pointer cursor
5. On the other browser, confirm you see the default cursor

### CODE REVIEW

**Comment by zenweasel** [comment]
Let's also remove the hover-state for images when you aren't an admin.

File: imports/plugins/included/default-theme/client/styles/media.less [hunk 1]
```diff
@@ -24,6 +24,10 @@
 .rui.media-gallery .gallery-image:hover {
   background-color: @list-group-hover-bg;
   opacity: 0.7;
+  // cursor: pointer;
```
[the remark this answers is not in the document]
Comment (codejockie) [reply imports/plugins/included/default-theme/client/styles/media.less]: Okay

**Comment by zenweasel** [comment]
I'm still getting a change in hover state when I mouseover the image as a non-admin (just that subtle change in size or border). We want nothing to happen when you mouseover the image.

image_mouseover

**Comment by codejockie** [comment]
Okay, will work on that.

**Comment by codejockie** [comment]
@zenweasel you can test it again, I believe it's fixed now.

**Comment by spencern** [comment]
`eslint` passes locally, bitHound doesn't reveal what is throwing the dependency issue. Merging.

**Pull request merged**